en/app.ftl

```diff
@@ -62,6 +62,19 @@ nav-profile-contact = Contact us
 nav-profile-contact-tooltip = Get in touch about { -brand-name-relay-premium }
 nav-profile-image-alt = { -brand-name-firefox-account(capitalization: "uppercase") } Avatar
 
+# Mobile menu text
```

Comment:
This should have ## or comment for a group of text. Other than that, the strings are good.

Reply:
@lloan Here are some details on the types of comments and where to use them:  https://projectfluent.org/fluent/guide/comments.html